### ISSUE TITLE
Not all export failures are detected

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#117]
Originally reported of Google Code with ID 61.

```
The export of an instance is done via a pipe (run the OS export command
piped to ssh to another node).

This means, due to the standard POSIX semantics, that failures of the OS
export script are not detected (bash can do this if a bash-specific option
is enabled, pipefail, but this is non-standard).

This in turn leads to exports failing silently. The QA suite also doesn't
detect this.

iustin
```
Originally added on 2009-07-29 13:21:12 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed in version 2.0.3.

thanks,
iustin
```
Originally added on 2009-08-08 18:26:51 +0000 UTC.

Added Labels: Component-node
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
